plugins/pipelines/app/models/concerns/pipeline_plugin_concern.rb

```diff
@@ -0,0 +1,33 @@
+module PipelinePluginConcern
+  # Return true if any stages in the pipeline are marked production
+  def production?
+    super || Stage.find(next_stage_ids).any? { |s| s.production? }
+  end
+
+  def next_stage
+    return Stage.find(next_stage_ids.first) unless next_stage_ids.empty?
+    stages = project.stages.to_a
+    stages[stages.index(self) + 1]
```

Comment:
This was the old function body. But yea, I can 'super' this like above now that the method overriding is working.